Gulpfile.mjs

```diff
@@ -222,7 +223,7 @@ function getFiles(glob, { include, exclude }) {
 }
 
 function createWorker(useWorker) {
-  const numWorkers = require("os").cpus().length / 2 - 1;
+  const numWorkers = Math.ceil(cpus().length / 2) - 1;
```

Comment:
`cpus().length` can be `0` so this will give you a negative number.
Ref https://github.com/yarnpkg/berry/pull/3090